python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -194,6 +194,12 @@ def __init__(
                 self.n_estimators_per_worker[i] + 1
             )
 
+        seeds = list()
+        seeds.append(0)
+        for i in range(1, len(self.n_estimators_per_worker)):
```

Comment:
Is this just the partial sums of `n_estimators_per_worker`? Maybe simpler to just do `seeds = itertools.accumulate(self.n_estimators_per_worker)` then

Reply:
Also, ideally, users could pass through their own random seed to control determinism at the Dask layer. Right now it seems like we'll always get the same seeds. This could be added in a follow-on PR.